### DISCUSSION TITLE
DHS-WIN-GDC-02 Garage Door Opener missing Relay 2

### DISCUSSION THREAD

**Discussion by zildfox** [zwave-js/zwave-js#7150] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
The DHS-WIN-GDC-02 Garage Door Opener only shows 1 of its 2 relays (shown as a binary switch)

What did you expect to happen instead?
Both Relays should be shown under "Binary Switch v1" (as long as parameter 9 is set to 0 - Do not hide Relay 2)

### Software versions

Driver (node-zwave-js): ... 13.2.0

Z-Wave JS UI: ... 9.18.0.5040e7e

### Device information

Manufacturer: ... zConnect
Model name: ... DHS-WIN-GDC-02
Node ID: ... 12

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[Untitled.log](https://github.com/user-attachments/files/16920966/Untitled.log)

[a comment by an automated account is not shown]

**Reply by zildfox** [reply]
[z-ui_2024-09-08.log](https://github.com/user-attachments/files/16920982/z-ui_2024-09-08.log)

**Reply by AlCalzone** [reply]
This is still not a driver log. Please read the information that's provided.

**Reply by zildfox** [reply]
Apologies - I re-read the instructions and attached a new log. Looks like it's the right syntax according to the examples and I included another device interview. Let me know if you need any further details, or a longer log (I wasn't sure how long to capture on debug mode). TIA :) 
[home-assistant_zwave_js_2025-04-28T07-48-08.202Z.log](https://github.com/user-attachments/files/19936257/home-assistant_zwave_js_2025-04-28T07-48-08.202Z.log)

**Reply by AlCalzone** [reply]
That looks better. Please re-interview the device while making the log and wait for that to finish first.

**Reply by zildfox** [reply]
Sorry I thought I had done that in the last log, here's another one. 
[Uploading home-assistant_zwave_js_2025-04-28T10-27-43.921Z.log…]()
